docs/source/recipes/index.rst

```diff
@@ -37,6 +37,13 @@ workflows.
 
 .. Add recipe cards below
 
+.. customcarditem::
+    :header: FiftyOne terminology
+    :description: A brief overview of FiftyOne's key terminology.
+    :link: fiftyone_terminology.html
+    :image: ../_static/images/recipes/pandas_vs_fiftyone.png
```

Comment:
Please add a new thumbnail image for this before merging! 

Tip: make sure you use a square image, or else the image will be squashed/stretched when it is displayed.

Reply:
yep! Just waiting on Michelle. She said she will get it in this weekend